api_docs/deploying-bots.md

```diff
@@ -36,6 +36,8 @@ Zulip Botserver starts a web server that listens to incoming messages
 from your main Zulip server. The sequence of events in a successful
 Botserver interaction are:
 
+{start_tabs}
```

Comment:
This section isn't instructions, but rather describing a sequence of events. If we want to keep these tabs as instruction blocks, maybe we shouldn't add them here.

Also, noting that the code highlighting for the json code block in the below bullet points is broken.

Reply:
Yeah, I agree. It's probably best to use tabs only for instruction blocks.

Yep, code highlighting is broken within a numbered list, and it looks like it can't be fixed just by tweaking the formatting so I removed it for now.

![image](https://github.com/zulip/zulip/assets/2343554/0063f609-e520-4691-90ad-d937ff01d704)

Reply:
Yeah, this looks better. Can we fix that final sentence and make it part of the 4th point?